src/test/java/org/folio/edge/patron/utils/PatronOkapiClientTest.java

```diff
@@ -213,7 +215,32 @@ public void testGetAccountOffsetIsWrong(TestContext context) throws Exception {
         resp -> {
           logger.info("mod-patron response body: " + resp.body());
           context.assertEquals(400, resp.statusCode());
-          context.assertEquals(wrongOffsetMessage, resp.bodyAsString());
+          context.assertEquals(String.format(WRONG_INTEGER_PARAM_MESSAGE, OFFSET, "-1"), resp.bodyAsString());
+          async.complete();
+        },
+        context::fail);
+    });
+  }
+
+  @Test
+  public void testGetAccountLimitIsWrong(TestContext context) throws Exception {
+    logger.info("=== Test getAccount - limit is wrong ===");
+
+    Async async = context.async();
+    client.login("admin", "password").thenAcceptAsync(v -> {
+      assertEquals(MOCK_TOKEN, client.getToken());
+
+      client.getAccount(PatronMockOkapi.patronId,
+        true,
+        true,
+        true,
+        null,
+        "-1",
+        null,
+        resp -> {
+          logger.info("mod-patron response body: " + resp.body());
+          context.assertEquals(400, resp.statusCode());
+          context.assertEquals(String.format(WRONG_INTEGER_PARAM_MESSAGE, LIMIT, "-1"), resp.bodyAsString());
```

Comment:
When tests re-use the definitions in the implementation e.g. the error message and parameter name, I think that can 
* obscure the expectations of the test e.g. what the message should be. This only states it should be the same as the implementation
* lead to false successes e.g. the limit constant is changed incorrectly in the implementation and this test still passes
* make it more difficult to use the test for a TDD style cycle where the expectations of the test change separately to the implementation

(I think this applies similarly to the changes to other tests in this pull request)

Reply:
Changed to use separate definitions for tests